petar-stoyanov/homework6/user-info.php

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>User Info</title>
+	<link rel="stylesheet" type="text/css" href="main.css">
+</head>
+<body>
+<header>
+	<h1>User Info</h1>
+</header>
+<main>
+	<?php
+		$fname="";
+		$lname="";
+		$pass="";
+		$email="";
+		$gender="";
+		$country="";
+		//assigning variables
+		if(isset($_POST['fname']) && isset($_POST['lname']) && isset($_POST['pass1']) && isset($_POST['pass2']) && isset($_POST['email1']) && isset($_POST['email2']) && isset($_POST['gender']) && isset($_POST['country'])) {
+			$fname = $_POST['fname'];
+			$lname = $_POST['lname'];
+			$gender = $_POST['gender'];
+			$country = $_POST['country'];
+		}
+		
+		//checks for incorrect form fields
+		if($_POST['pass1'] != $_POST['pass2']) {
```

Comment:
@petar-stoyanov7 
It is good practice to check whether the $_POST parameters are set before using them : )